docs/source/user_guide/accessing_results.rst

```diff
@@ -110,12 +112,13 @@ The stress ``Result`` object can be obtained as follows:
     >>> from ansys.dpf.post import examples
     >>> solution = post.load_solution(examples.multishells_rst)
 
-    Instantiate the stress result object 
+    Instantiate the stress result object
```

Comment:
```suggestion
    # Instantiate the stress result object
```

Reply:
This change applies everywhere, I've already seen it multiple times. Especially now since you added the copybutton, it is important that code comments are rendered as such inside the ``code`` blocks. Otherwise Python will try to interpret them.